### PR TITLE
how to use a per document monitor - Alerting plugin

### PR DESCRIPTION
**Pull request by alicejw-aws**
Signed-off-by: alicejw <alicejw@amazon.com>

### Description
Write how to create and use the new per document monitor as part of the OpenSearch Dashboards Alerting plugin.

### Issues Resolved
[475](https://github.com/opensearch-project/documentation-website/issues/475)
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _monitoring-plugins/alerting/index.md [hunk 1]
```diff
@@ -11,7 +11,11 @@ redirect_from:
 OpenSearch Dashboards
 {: .label .label-yellow :}
 
-The alerting feature notifies you when data from one or more OpenSearch indices meets certain conditions. For example, you might want to notify a [Slack](https://slack.com/) channel if your application logs more than five HTTP 503 errors in one hour, or you might want to page a developer if no new documents have been indexed in the past 20 minutes.
+OpenSearch Dashboards provides the Alerting plugin that allows you to monitor your data and create notifications that trigger when conditions occur in one or more indexes.
```
Comment (keithhc2) [inline _monitoring-plugins/alerting/index.md]: Can we rephrase this to "You can use the Alerting plugin in OpenSearch Dashboards to monitor your data and create notifications..."?

File: _monitoring-plugins/alerting/monitors.md [hunk 2]
```diff
@@ -181,7 +199,7 @@ Whereas query-level monitors run your specified query and then check whether the
     **Note**: Anomaly detection is available only if you are defining a per query monitor.
     {: .note}
 
-1. Choose a frequency and timezone for your monitor. Note that you can only pick a timezone if you choose Daily, Weekly, Monthly, or [custom cron expression]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron/) for frequency.
+1. Choose a frequency to run your monitor. You can run it either by time intervals (minutes, hours, or days), or on a schedule. If you run it on a schedule such as daily, weekly, monthly, or [custom cron expression]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron/), then you need to also provide the timezone.
```
Comment (keithhc2) [inline _monitoring-plugins/alerting/monitors.md:202]: Should timezone be one word?

File: _monitoring-plugins/alerting/index.md [hunk 3]
```diff
@@ -11,7 +11,11 @@ redirect_from:
 OpenSearch Dashboards
 {: .label .label-yellow :}
 
-The alerting feature notifies you when data from one or more OpenSearch indices meets certain conditions. For example, you might want to notify a [Slack](https://slack.com/) channel if your application logs more than five HTTP 503 errors in one hour, or you might want to page a developer if no new documents have been indexed in the past 20 minutes.
+OpenSearch Dashboards provides the Alerting plugin that allows you to monitor your data and create alert notifications that trigger when conditions occur in one or more indexes.
+
+You create a monitor with trigger conditions that generate various alert notifications through the message channel you select as a destination. Notifications can be sent to email, Slack, or Amazon Chime.
+
+The monitor you create notifies you when data from one or more OpenSearch indices meets certain conditions. For example, you might want to notify a [Slack](https://slack.com/) channel if your application logs more than five HTTP 503 errors in one hour, or you might want to page a developer if no new documents have been indexed in the past 20 minutes.
```
Comment (lizsnyder) [inline _monitoring-plugins/alerting/index.md:18]: indexes

File: _monitoring-plugins/alerting/monitors.md [hunk 4]
```diff
@@ -97,15 +115,15 @@ POST _nodes/reload_secure_settings
...
+Per query monitors run your specified query and then check whether the query's results triggers any alerts. Per bucket monitors let you select fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time.
 
-1. Define the monitor in one of three ways: visually, using a query, or using an anomaly detector.
+1. Decide how you want to define your query and triggers. You can use either of three methods: visual editor, query editor, or anomaly detector.
```
Comment (lizsnyder) [inline _monitoring-plugins/alerting/monitors.md:126]: Maybe "any of the following" instead of "either". Either seems more appropriate for if there are only 2 options.

File: _monitoring-plugins/alerting/monitors.md [hunk 5]
```diff
@@ -202,6 +220,15 @@ The line moves up and down as you increase and decrease the threshold. Once this
...
 Bucket-level monitors also require you to specify a threshold and value for your aggregation and timeframe, but you can use a maximum of five conditions to better refine your trigger. Optionally, you can also use a keyword filter to filter for a specific field in your index.
 
+Document-level monitors provide the added option to use tags that represent multiple queries connected by the logical OR operator.
+
+To create a multiple query combination trigger:
+
+1. Create a per document monitor with more than one query.
+2. Create the first query with field, operator and value. For example, set the query to search for the `region` field with either operator: "is" or "is not", and set the value "us-west-2".)
+3. Select **Add Tag** and give it a name.
+3. Create the second query add the same tag to it.
```
Comment (lizsnyder) [inline _monitoring-plugins/alerting/monitors.md:230]: and?

**Review by lizsnyder — APPROVED** [review]

File: _monitoring-plugins/alerting/index.md [hunk 6]
```diff
@@ -11,7 +11,11 @@ redirect_from:
 OpenSearch Dashboards
 {: .label .label-yellow :}
 
-The alerting feature notifies you when data from one or more OpenSearch indices meets certain conditions. For example, you might want to notify a [Slack](https://slack.com/) channel if your application logs more than five HTTP 503 errors in one hour, or you might want to page a developer if no new documents have been indexed in the past 20 minutes.
+OpenSearch Dashboards provides the Alerting plugin that allows you to monitor your data and create alert notifications that trigger when conditions occur in one or more indexes.
```
Comment (keithhc2) [inline _monitoring-plugins/alerting/index.md:14]: Can we rephrase this to "You can use OpenSearch Dashboards with the Alerting plugin to monitor your data and create notifications that trigger..."?

**Review by keithhc2 — APPROVED** [review]

**Comment by keithhc2** [comment]
@alicejw-aws Is there an API component to this? Can users use the API to create per query monitors?

**Comment by alicejw-aws** [comment]
> @alicejw-aws Is there an API component to this? Can users use the API to create per query monitors?

yes will add API section and submit for review again.

File: _monitoring-plugins/alerting/monitors.md [hunk 7]
```diff
@@ -202,6 +220,15 @@ The line moves up and down as you increase and decrease the threshold. Once this
...
 
+Document-level monitors provide the added option to use tags that represent multiple queries connected by the logical OR operator.
+
+To create a multiple query combination trigger:
+
+1. Create a per document monitor with more than one query.
+2. Create the first query with field, operator and value. For example, set the query to search for the `region` field with either operator: "is" or "is not", and set the value "us-west-2".)
+3. Select **Add Tag** and give it a name.
+3. Create the second query and add the same tag to it.
+4. Now you can create the trigger condition and specify the tag name. This creates a combination trigger that checks two queries that both contain the same tag. The monitor checks both queries by a logical OR operation if either query's conditions are met, then it will generate the alert notification.
```
Comment (natebower) [inline _monitoring-plugins/alerting/monitors.md:231]: Third sentence: Should "by a logical OR operation" be "with a logical OR operation"? The sentence also doesn't connect grammatically. Should it be "The monitor checks both queries with a logical OR operation, and if either..."?

File: _monitoring-plugins/alerting/monitors.md [hunk 8]
```diff
@@ -202,6 +220,15 @@ The line moves up and down as you increase and decrease the threshold. Once this
 
 Bucket-level monitors also require you to specify a threshold and value for your aggregation and timeframe, but you can use a maximum of five conditions to better refine your trigger. Optionally, you can also use a keyword filter to filter for a specific field in your index.
 
+Document-level monitors provide the added option to use tags that represent multiple queries connected by the logical OR operator.
+
+To create a multiple query combination trigger:
```
[the remark this answers is not in the document]
Comment (alicejw-aws) [reply _monitoring-plugins/alerting/monitors.md:225]: we should discuss in our style guide meeting. this is a common construct. "to <task> :" followed by procedural steps.

[the remark this answers is not in the document]
Comment (natebower) [reply _monitoring-plugins/alerting/monitors.md]: Agreed that we should discuss. If treating this as a procedure _title_, the "To" phrase should not include ending punctuation. If treating as an introduction to an ordered (numbered) list, we should use a sentence that ends with a colon.

[the remark this answers is not in the document]
Comment (alicejw-aws) [reply _monitoring-plugins/alerting/monitors.md]: let me provide some examples for you, this is a common construct. thanks

[the remark this answers is not in the document]
Comment (natebower) [reply _monitoring-plugins/alerting/monitors.md]: Agreed, it's definitely a common construct. We just need to discuss the style to be followed. Let's discuss in our next meeting so we can determine a consistent standard.
